wolfmqtt_trust_platform/firmware/src/third_party/wolfMQTT/src/mqtt_socket.c

```diff
@@ -27,7 +27,7 @@
 #ifdef WOLFMQTT_NONBLOCK
     /* need EWOULDBLOCK and EAGAIN */
     #ifdef MICROCHIP_MPLAB_HARMONY
-        #include <sys/errno.h>
+        #include <errno.h>
```

Comment:
I didn't realize that we had a copy of the wolfMQTT code here... @jackctj117 please test with the latest wolfMQTT and wolfSSL. This wolfMQTT issue has already been fixed... see here -> https://github.com/wolfSSL/wolfMQTT/blob/master/src/mqtt_socket.c#L30

Reply:
I see, I didn't realize the demo was using an older version. I am going to try again with all updated wolfMQTT and wolfSSL files.